### PR TITLE
#219 Domain - Add new Aggregate: AnnouncementAggregate.cs + Propertie…

### LINKED ISSUE #219: Create Project Announcements
I would like to be able to be able to create Announcements on any Project.

This implementation is similar for the **Project Sprint Aggregate,** therefore, I _recommend_ to take a quick look on that, in parallel, to get an overall idea.
It is also _recommended_ that you check out the **Feature Development Workflow** documentation, as you work on this new feature.

1. Create the Announcement **Aggregate**. The announcement should have the following fields:
    - Title
    - Description
    - Audit
    - Project (nullable)
    - Workspace (nullable)
    - 1 create method that takes title, description, user for audit and project as params
2. Create **Properties Validation** for the new aggregate
3. Create its Persistence **Configuration**
4. Create Migration (I can do it quick, or you can read the documentation on **Database**) 
5. Create its Persistence **Repository** (+ interfaces on App layer)
6. Create its Create **Command** with Handler, Result and Validator (Recommended command name: **CreateAnnouncementCommand**). 
    - possible fields on Command: 
      - project id, 
      - title string 
      - and description
    - possible validations: 
      - title not empty and longevity, 
      - description not empty
      - project id not empty
    - handler ideas:
      - possible errors: Project Not Found
      - use the ICurrentUserProvider to get the current user for the audit
7. Create its Endpoint with a new controller, new contracts + mapping config
8. Create Unit Tests for Application and Api layers (Copy Paste another one and modify it, especially for the controller)

### PR DESCRIPTION
**Pull request by CristinaNeacsu**
…sValidation.Announcement.cs

### CODE REVIEW

File: src/ConsiliumTempus.Infrastructure/Persistence/Repository/AnnouncementRepository.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+﻿using ConsiliumTempus.Application.Common.Interfaces.Persistence.Repository;
+using ConsiliumTempus.Domain.Announcement;
+using ConsiliumTempus.Infrastructure.Persistence.Database;
+
+namespace ConsiliumTempus.Infrastructure.Persistence.Repository;
+
+public class AnnouncementRepository(ConsiliumTempusDbContext dbContext) : IAnnouncementRepository
```
Comment (mihai2000q) [inline src/ConsiliumTempus.Infrastructure/Persistence/Repository/AnnouncementRepository.cs:7]: Should have been a sealed class (this is will also solve the Architecture Tests failure, also ignore the unit tests, that's a false negative)
